app/controllers/planning_applications/neighbour_letters_controller.rb

```diff
@@ -114,7 +118,7 @@ def require_reason_when_resending
     end
 
     def update_consultation!
-      @consultation.update!(consultation_params.except(:resend_existing, :resend_reason).merge(status: "in_progress"))
+      @consultation.send_neighbour_emails(consultation_params.except(:resend_existing, :resend_reason).merge(status: "in_progress"))
```

Comment:
I find this kind of confusing - a method that calls update_! actually does another action that doesn't really seem to have anything to do with updating. But then the update is in that action? And send_neighbour_emails doesn't seem to send anything? 

It seems like the methods don't do what I'm expecting them to

Reply:
I named it to be consistent with the `send_consultee_emails` method but you're right that it doesn't necessarily make sense to be called that. (And anyway should be letters, not emails.)

I originally had the logic just in `update_consultation!` but that seemed to require giving it too much knowledge of a consultation — e.g., if the consultation hasn't started yet, it must be started or `extend_deadline` will fail.

I think if I move the `start_deadline` into `extend_deadline` then the `update!` and `extend_deadline!` can be called from `update_consultation!`.